README.md

```diff
@@ -837,6 +837,137 @@ OPTIONS:
 --timestamps, -t           [Optional] Shows timestamps on each line in the log output.
 ```
 
+## Using Private Registry Authentication
+
+If you want to use privately hosted container images with ECS, the ECS CLI can store your private registry credentials for you with AWS Secrets Manager and create an IAM role which ECS can use to access the credentials and private images. This allows you to:
+
+* Store private registry credentials within AWS for use with ECS or other services
```

Comment:
Are we sure we can say "or other services"? The secret format that we use is specific to ECS, I am not sure there is any guarantee it will necessarily work for other services.